react/getting_started_with_react/keys_in_react.md

```diff
@@ -57,22 +57,22 @@ function TodoList() {
 }
 ~~~
 
-<span id="index-as-key">Additionally, if the list is static and will remain unchanged throughout the application's life, we can use the array index as a key. Though you should be careful when working with `index` as key.</span>
+<span id="index-as-key">Additionally, if the list is static and will remain unchanged throughout the application's life, we can use the array index as a key. Though you should be careful when working with `index` as a key.</span>
```

Comment:
Should we expand on the why here?

Reply:
I was just going through this lesson and agree it would be nice to add "why we should be careful when working with `index` as a key"

Reply:
what do you reckon @xandora ?

Reply:
@Eveieve did you see the above? 

Some notes: 
- the headers should all be fixed already with the NodeJS path being fully updated to sentence-case last week
- I think it's good to proceed with the above change; add a note that the `why` will be covered in the assignment section. 
- there's a merge conflict to look into before we can merge it :)

Reply:
@ManonLef Hi, I left a comment above but I think there was a problem with this 'pending' next to my name. I think that's what's causing the merge conflict. 

![image](https://github.com/TheOdinProject/curriculum/assets/85281165/0871975a-a5ef-4284-9749-f9582ccee972)

I will post my suggestion here again to answer your second question: 

```suggestion
<span id="index-as-key">Additionally, if you're sure the list will remain unchanged throughout the application's life,  you can use the array index as a key. But this is [not recommended](https://react.dev/learn/rendering-lists#why-does-react-need-keys) since it can lead to confusing bugs if the list *does* change with any items deleted, inserted or rearranged. 
```
To your first question: I didn't change any headers for this PR, did you mean to just let me know about it?

Reply:
I think I mentioned it because I asked you to change them earlier somewhere in this PR. 
If it had `pending` that's probably a review you started but didn't submit :) 

When I referred to `the above` your `pending` wasn't there yet so I meant the discussion me and Xandora had about it:

![Screenshot 2023-07-31 at 20 42 24](https://github.com/TheOdinProject/curriculum/assets/81025586/ba8f272b-7154-44b8-b538-736022d327aa)

The resource is shared later in the lesson so linking to it here as well doesn't feel like the best thing to do.

Reply:
I like it! I'm just not sure about the last sentence:

> More on why this is will be covered in the reading later.

How do you feel about:

> You will learn more about this in the assignment section's linked articles. 

?

Reply:
You can commit your own actually. I will do it now and do a final suggestion. Sorry about that